### ISSUE TITLE
Adding doas support.

### ISSUE THREAD

**Issue by Scupake** [scop/bash-completion#505]
Can someone please add support for ```doas```?

**Comment by scop** [comment]
Have you asked if upstream doas project would be interested int creating or maintaining one?

**Comment by Scupake** [comment]
No, I haven't, I guess I could open an issue for [OpenDoas](https://github.com/Duncaen/OpenDoas), which is the version used in Void, Debian, Ubuntu and Gentoo.

**Comment by chunzu** [comment]
> Can someone please add support for `doas`?

Just copy `sudo` file from `completions` directory and rename it to `doas`. Then search and replace all occurrences of `sudo` in this file with `doas`.
Wait for a proper solution :)

**Comment by fethomm** [comment]
That helps some, but not with completion for apt in Debian. Do you have an idea how to work around that?

**Comment by chunzu** [comment]
Hello. I'm on Debian as well. Completion for apt commands work fine.

This is my /etc/doas.conf:
`permit keepenv nopass USER as root`

Without "keepenv" you can't start GUI applications. But even with this, there are some errors about DBUS and others. So please do not launch GUIs as it can break your system.

Also on a second line of **doas** completion file, add this:
`PATH=$PATH:/sbin:/usr/sbin:/usr/local/sbin`

To fix "command not found" error for programs in /usr/sbin.

Sorry, I can not help you more than this. We have to wait when somebody implements it.

**Comment by fethomm** [comment]
Thanks, that helped!

**Comment by Scupake** [comment]
Opendoas' maintainer has said in a [reddit comment](https://www.reddit.com/r/debian/comments/lsv97d/doas_a_minimal_sudo_alternative_is_now_packaged/gp828g1/?context=3) that they won't add any feature unless it is in upstream doas, which is unlikely to add bash support because iirc they use ksh on openbsd

**Comment by scop** [comment]
OpenBSD has bash-completion in their ports tree, https://openports.se/shells/bash-completion

**Comment by akinomyoga** [comment]
Ref. PR https://github.com/Duncaen/OpenDoas/pull/84 in OpenDoas
